grails-app/services/com/unifina/service/CanvasService.groovy

```diff
@@ -97,6 +97,7 @@ class CanvasService {
 
 		Map signalPathContext = canvas.toMap().settings
 
+		canvas.startedBy = asUser
```

Comment:
Can this be moved into `SignalPathService#startLocal`? Feels like it belongs there since there is a bunch of other similar statements there that update the contents of the Canvas domain object.